todo_app/app.py

```diff
@@ -45,10 +51,19 @@ def unauthenticated():
 
     @login_manager.user_loader
     def load_user(user_id):
+        app.logger.info(f"User is authenticated and their role is {User(user_id).user_role}, with ID of {user_id}")        
         return User(user_id)
 
     login_manager.init_app(app)
 
+    if app.config['LOGGLY_TOKEN'] is not None:
+        handler = HTTPSHandler(f'https://logs-01.loggly.com/inputs/{app.config["LOGGLY_TOKEN"]}/tag/todo-app')
+        handler.setFormatter(jsonlogger.JsonFormatter("[%(asctime)s] %(levelname)s in %(module)s: %(message)s"))
+
+        getLogger('werkzeug').addHandler(HTTPSHandler(f'https://logs-01.loggly.com/inputs/{app.config["LOGGLY_TOKEN"]}/tag/todo-app-requests'))
```

Comment:
Nice work on this extension bit - I can see those requests coming through no issue 👍